lib/cli/src/commands/compile.rs

```diff
@@ -54,7 +54,7 @@ impl Compile {
                 wasmer_engine_object_file::ObjectFileArtifact::get_default_extension(target_triple)
             }
             #[cfg(not(all(feature = "native", feature = "jit", feature = "object-file")))]
-            _ => bail!("selected engine type is not compiled in"),
+            _ => panic!("selected engine type is not compiled in"),
```

Comment:
Since it’s done on the CLI level, and the error is kind of expected I believe bail is better suited.

Reply:
For context, the function does not return `anyhow::Result` so we'd have to update that and all the call locations, which may be fine, I just did the smallest change to get it to compile.

Reply:
I agree though that `bail` is better, all things being equal -- I'll see if making it `anyhow::Result` is simple